doc/Settings.md

```diff
@@ -354,25 +354,3 @@ You can enable the feature as shown below.
        "resume": true
    },
 ```
-
```

Comment:
Not related to your change, but I noticed that `Settings.md` is outdated because it lists `resume` but not `fonts`